### ISSUE TITLE
job_dispatch should indicate whether or not stdout/stderr files exists

### ISSUE THREAD

**Issue by jondequinor** [equinor/libres#815]
`status.json` does not guarantee that the `stderr`/`stdout` paths of the job it represents exists.

We infrequently update `status.json`, so checking for the existence of these files should not be an unaffordable luxury.

**Comment by markusdregi** [comment]
Forgive me for playing the devil's advocate here, but are we sure this is problematic and something we should handle? The api that serves the stdout and stderr should definitively be robust towards this, but do we need the implementation below that to deal with this?

**Comment by jondequinor** [comment]
If we were not building software on top of this API, it wouldn't be problematic.

The benefit of a local solution (i.e. `job_dispatch`), is that you don't have to do `stat` calls in e.g. the GUI in order to verify the existence of these files.

**Comment by markusdregi** [comment]
I agree that this should be abstracted away before hitting the GUI. But whether the file should always exist or whether the API for instance could serve an empty stream if the file does not exist, that I'm more unsure of.
